inc/config_store.h

```diff
@@ -79,6 +79,12 @@ typedef struct ConfigStore {
     char *_replica_path;
 } ConfigStore;
 
+/// <summary>
+/// Delete the left over temp files during the device power up
+/// </summary>
+void ConfigStore_DeleteAllTempFiles(char*);
```

Comment:
What's the parameter's name and description?

Reply:
Added the parameter's description and name.